packages/zmarkdown/client/index.js

```diff
@@ -0,0 +1,51 @@
+export const plugins = {}
+export let defaultType
+
+export function use (obj) {
+  if (!obj) {
+    throw new Error('This is not an object')
+  }
+
+  // Check the object structure
+  if (!('name' in obj)) {
```

Comment:
Don't use `in` to check for the presence of a key, prefer `hasOwnProperty` (same for the rest of the file)

Reply:
Can you please explain why use `hasOwnProperty` instead of `in` ?

Reply:
It's more precise and corresponds better to what we want to achieve here. We don't care about a method `obj.name()` (i.e. `YourObjectsPrototype.prototype.name`) for instance, what we're looking for is a property.